DemoApp/ChatPresenter.swift

```diff
@@ -21,6 +21,7 @@ extension UIViewController {
         // Config
         var uiConfig = UIConfig<NoExtraData>()
         uiConfig.navigation.channelListRouter = DemoChatChannelListRouter.self
+        UIConfig.default = uiConfig
```

Comment:
there three lines are not correct. It should be just:
```swift
UIConfig.default.navigation.channelListRouter = DemoChatChannelListRouter.self
```

Reply:
Done.